certified-connectors/SmartCOMM Doc-Gen/intro.md

```diff
@@ -0,0 +1,115 @@
+Smart Communications™ is a leading cloud-based platform for enterprise customer communications. As the only cloud solution ranked as a Leader in Gartner’s Magic Quadrant for Customer Communications Management, more than 350 global brands – many in the world’s most highly regulated industries – rely on Smart Communications to make multi-channel customer communications more meaningful, while also helping them simplify their processes and operate more efficiently.
```

Comment:
@sc-dnagy please remove the intro.md from the PR as it is not required in GitHub. you can submit this in ISV Studio once we approve this PR.